clients/src/main/java/org/apache/kafka/clients/consumer/internals/ClassicKafkaConsumer.java

```diff
@@ -103,11 +103,15 @@
  *
  * <p/>
  *
- * <em>Note:</em> per its name, this implementation is left for backward compatibility purposes. The updated consumer
- * group protocol (from KIP-848) introduces allows users continue using the legacy "classic" group protocol.
- * This class should not be invoked directly; users should instead create a {@link KafkaConsumer} as before.
+ * This {@link ConsumerDelegate} implementation exists for backward compatibility to allow users to continue to use
+ * the existing classic group protocol.
```

Comment:
I think "existing" is redundant here. I suggest "classic group protocol (pre-KIP 848)".

Reply:
Fixed, as suggested. Thanks!